lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -99,7 +99,8 @@ describe('PerfCategoryRenderer', () => {
     assert.ok(oppSparklineElement.title, 'did not set tooltip on sparkline');
   });
 
-  it('renders the performance opportunities with a debug string', () => {
+  // TODO(phulce): verify these don't happen anymore
+  it.skip('renders the performance opportunities with a debug string', () => {
```

Comment:
in a branch of mine, i handle this case.

Reply:
cool beans

Reply:
yes indeed, removed